src/routers/CancelMatchRouter.ts

```diff
@@ -3,7 +3,7 @@ import Constants from '../common/constants';
 import MatchRepo from '../repos/MatchRepo';
 import AuthenticatedApplicationRouter from '../utils/AuthenticatedApplicationRouter';
 
-class UpdateMatchAvailabilitiesRouter extends AuthenticatedApplicationRouter<void> {
+class CancelMatchRouter extends AuthenticatedApplicationRouter<void> {
   constructor() {
```

Comment:
Whats the reason for the name change?

Reply:
The name was supposed to be CancelMatchRouter from the start, the internal class name was just left unchanged. It used to be UpdateMatchAvailabilitiesRouter because initially Manish and I thought the only update we would make is setting an availability to canceled but we would need to update it whenever the user suggests certain times, etc.